### PR TITLE
Remove IniFile from spawn command

### PR DESCRIPTION
**Pull request by bearbin**
Spiritual replacement for #172, which is broken due to merge conflicts.

Not tested, requires thorough checking, although it is really only minor modifications of existing code.

Also added CONTRIBUTORS file and updated license to standard form.

### CODE REVIEW

**Comment by bearbin** [comment]
The failed build doesn't appear to be related to this code.

File: spawn.lua [hunk 1]
```diff
@@ -67,23 +65,18 @@ end
...
-	WorldIni:SetValueI("SpawnPosition", "Z", PlayerZ)
-	WorldIni:WriteFile(Player:GetWorld():GetIniFileName())
-
-	SendMessageSuccess(Player, string.format("Changed spawn position to [X:%i Y:%i Z:%i]", PlayerX, PlayerY, PlayerZ))
-	return true
+	if ( World:SetSpawn(PlayerX + 0.5, PlayerY, PlayerZ + 0.5) ) then
+		SendMessageSuccess(Player, string.format("Changed spawn position to [X:%i Y:%i Z:%i]", PlayerX, PlayerY, PlayerZ))
+		return true
+	else
+		SendMessageFaulure(Player, "Spawn setting failed.")
```
Comment (mathiascode) [inline spawn.lua]: Typo. Faulure -> Failure

**Comment by bearbin** [comment]
Fixed typo.

File: spawn.lua [hunk 2]
```diff
@@ -67,23 +65,18 @@ end
...
-	WorldIni:DeleteValue("SpawnPosition", "Z")
-
-	WorldIni:SetValueI("SpawnPosition", "X", PlayerX)
-	WorldIni:SetValueI("SpawnPosition", "Y", PlayerY)
-	WorldIni:SetValueI("SpawnPosition", "Z", PlayerZ)
-	WorldIni:WriteFile(Player:GetWorld():GetIniFileName())
-
-	SendMessageSuccess(Player, string.format("Changed spawn position to [X:%i Y:%i Z:%i]", PlayerX, PlayerY, PlayerZ))
-	return true
+	if ( World:SetSpawn(PlayerX + 0.5, PlayerY, PlayerZ + 0.5) ) then
```
Comment (madmaxoft) [inline spawn.lua]: Why the `floor` and `+ 0.5`? Can't we just leave the position as-is?

Comment (bearbin) [inline spawn.lua]: I don't know, this is copied from #172, which you approved.

Comment (Bond-009) [inline spawn.lua]: I like it, it auto-corrects to the middle of the block.

Comment (madmaxoft) [inline spawn.lua]: @bearbin Didn't occur to me then
@Bond-009 Exactly, that could be a problem if you want to have a spawn on top of an open trapdoor above a deep abyss, for example. I think in this case we don't want to "fix" what ain't broken.

**Comment by williamhatcher** [comment]
And isn't that the fun of Minecraft? Falling over and over again, reminding you of your failures!

**Comment by bearbin** [comment]
The floor has been removed and precise coordinates passed through.

**Comment by madmaxoft** [comment]
The CI check is a problem in the Plugin Checker.

**Pull request merged**